mev_inspect/aave_liquidations.py

```diff
@@ -51,7 +51,17 @@ def get_aave_liquidations(
                 trace.transaction_hash, trace.trace_address, traces
             )
 
-            received_amount = _get_liquidator_payback(child_traces, liquidator)
+            payback_transfer = _get_liquidator_payback(child_traces, liquidator)
+
+            if payback_transfer:
+                assert isinstance(payback_transfer, ERC20Transfer)
+                received_amount = payback_transfer.amount
+                received_token_address = payback_transfer.token_address
```

Comment:
your favorite :) 

if someone changes one of these branches to not set `received_token_address` then it becomes an awkward variable does not exist down below

Reply:
True, done